admin/load.php

```diff
@@ -126,31 +126,44 @@ function perflab_render_modules_page() {
  * @param array  $module_settings Associative array of the module's current settings.
  */
 function perflab_render_modules_page_field( $module_slug, $module_data, $module_settings ) {
-	$base_id   = sprintf( 'module_%s', $module_slug );
-	$base_name = sprintf( '%1$s[%2$s]', PERFLAB_MODULES_SETTING, $module_slug );
-	$enabled   = isset( $module_settings['enabled'] ) && $module_settings['enabled'];
+	$base_id         = sprintf( 'module_%s', $module_slug );
+	$base_name       = sprintf( '%1$s[%2$s]', PERFLAB_MODULES_SETTING, $module_slug );
+	$enabled         = isset( $module_settings['enabled'] ) && $module_settings['enabled'];
+	$can_load_module = perflab_can_load_module( $module_slug );
 	?>
 	<fieldset>
 		<legend class="screen-reader-text">
 			<?php echo esc_html( $module_data['name'] ); ?>
 		</legend>
 		<label for="<?php echo esc_attr( "{$base_id}_enabled" ); ?>">
-			<input type="checkbox" id="<?php echo esc_attr( "{$base_id}_enabled" ); ?>" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>" aria-describedby="<?php echo esc_attr( "{$base_id}_description" ); ?>" value="1"<?php checked( $enabled ); ?>>
-			<?php
-			if ( $module_data['experimental'] ) {
-				printf(
-					/* translators: %s: module name */
-					__( 'Enable %s <strong>(experimental)</strong>', 'performance-lab' ),
-					esc_html( $module_data['name'] )
-				);
-			} else {
-				printf(
-					/* translators: %s: module name */
-					__( 'Enable %s', 'performance-lab' ),
-					esc_html( $module_data['name'] )
-				);
-			}
-			?>
+			<?php if ( ! $can_load_module ) { ?>
+				<input type="checkbox" id="<?php echo esc_attr( "{$base_id}_enabled" ); ?>" disabled>
+				<input type="hidden" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>">
```

Comment:
This needs the `value` attribute now I think, but only _if_ that module is currently active via the option.

We need to make sure that a module that is configured as active in the DB remains active when updating any other modules on this page, even when it can't be loaded. The same for when a module is inactive, it should remain inactive.

For example like this:
```suggestion
				<input type="hidden" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>" value="<?php echo $enabled ? '1' : ''; ?>">
```

Reply:
Thanks. It makes sense to me. The review was implemented in PR.